pipeline/config.go

```diff
@@ -25,6 +25,15 @@ func (c Config) BuildPipeline(context operator.BuildContext) (*Pipeline, error)
 		return nil, err
 	}
 
+	if defaultOutput != nil {
+		for _, op := range operators {
+			if op.CanOutput() && len(op.Outputs()) == 0 {
+				op.SetOutputs([]operator.Operator{defaultOutput})
```

Comment:
The intent is that any operator that is missing an output will get the default one.

This will simplify configuration for OpenTelemetry users, who will be able to focus on the ingestion and processing that they want, and avoids forcing them to include a mysterious operator in every configuration.

Reply:
Additionally, the intent is not to change any other functionality. That is, omitting an operator in most cases will still result in the assumption that the next operator is the output. This really only applies to the last operator.

Reply:
Thanks for clarifying. In that case, I don't think this is quite doing what you're intending. The semantics of `SetOutputs` is a little confusing. Rather than literally setting the outputs of the operator, `SetOutputs` tells the operator "These are the operators that are available. You know which operator IDs you're looking for, so find them and save a reference to them". You can find the implementation here: https://github.com/observIQ/stanza/blob/e010fa6724e853ba45796fb78f86647121cf3c79/operator/helper/writer.go#L77

I think for this to work, you'd need to also set the configured outputs on the last operator in the pipeline to the ID of the default output.

Reply:
Are you sure? It seems to work as intended, both in unit tests and in the OT collector.

Reply:
Interesting...taking a closer look now

Reply:
It looks like it's working because the output is being appended to the end of the pipeline, and the last operator (now second-to-last) will output to the next operator by default. I'm able to delete the for loop entirely and have all the tests still pass. I don't have my the OT collector set up right now -- could you validate that it works there too? 

https://github.com/observIQ/stanza/pull/102